controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -4,13 +4,15 @@
 
 from referee.consts import MATCH_TIME, TIME_STEP
 from referee.referee import RCJSoccerReferee
+from recorder.recorder import VideoRecordAssistant
 
+automatic_mode = False
```

Comment:
Set `True` to automatically start recording with FAST mode. And quit Webots when the video exporting is done.

Reply:
@aquahika Cool! I think we can also make this listen to the presence of some env variable. As you suggested, it will be probably a bit difficult to set up but since we'll run this from Docker anyway, it should not be such a big deal -- that env variable will be exported for the whole container.